inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java

```diff
@@ -0,0 +1,255 @@
+                .append(" FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = '")
+                .append(schemaName)
+                .append("' AND TABLE_NAME = '")
+                .append(tableName)
+                .append("' ;");
+        LOGGER.info("check table sql: {}", sqlBuilder);
+        return sqlBuilder.toString();
+    }
+
+    /**
+     * Build SQL to check whether the column exists.
+     *
+     * @param schemaName SQLServer schema name
+     * @param tableName SQLServer table name
+     * @param columnName SQLServer column name
+     * @return the check column SQL string
+     */
+    public static String getCheckColumn(String schemaName, String tableName, String columnName) {
+        StringBuilder sqlBuilder = new StringBuilder();
+        sqlBuilder.append("SELECT COUNT(1) ")
+                .append(" FROM  INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA='")
+                .append(schemaName)
+                .append("' AND TABLE_NAME = '")
+                .append(tableName)
+                .append("' AND COLUMN_NAME = '")
+                .append(columnName)
+                .append("';");
+        LOGGER.info("check table sql: {}", sqlBuilder);
+        return sqlBuilder.toString();
+    }
+
+    /**
+     * Build SQL to check whether the schema exists.
+     *
+     * @param schemaName
+     * @return
+     */
+    public static String getCheckSchema(String schemaName) {
+        StringBuilder sqlBuilder = new StringBuilder();
+        sqlBuilder.append("SELECT COUNT(1) FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME ='")
+                .append(schemaName)
+                .append("';");
+        LOGGER.info("check schema sql: {}", sqlBuilder);
+        return sqlBuilder.toString();
+    }
+
+    /**
+     * Build create schema SQL.
+     *
+     * @param schemaName SQLServer schema name
+     * @return
+     */
+    public static String buildCreateSchemaSql(String schemaName) {
+        return new StringBuilder()
+                .append("CREATE SCHEMA \"")
+                .append(schemaName)
+                .append("\" AUTHORIZATION dbo ;")
+                .toString();
+    }
+
+    /**
+     * Build create table SQL by SQLServerTableInfo.
+     *
+     * @param table SQLServer table info {@link SQLServerTableInfo}
+     * @return the create table SQL String
+     */
+    public static List<String> buildCreateTableSql(SQLServerTableInfo table) {
+        List<String> sqls = new ArrayList<>();
+        StringBuilder sql = new StringBuilder();
+        // Support _ beginning with underscore
+        sql.append("CREATE TABLE ").append(table.getSchemaName())
+                .append(".")
+                .append(table.getTableName());
+
+        // Construct columns and partition columns
+        sql.append(buildCreateColumnsSql(table));
+        sqls.add(sql.toString());
+        table.getColumns().stream().filter(column -> StringUtils.isNotEmpty(column.getComment()))
```

Comment:
Personally, I think it is better to change the filter operation to a new line.

Reply:
OK